public/js/directives/years/years-directive.html

```diff
@@ -6,10 +6,10 @@
             <h3 ng-if="::!readonly" contenteditable ng-model="year.title"></h3>
             <h3 ng-if="::readonly">{{::year.title}}</h3>
         </div>
-        <button class="btn btn-default btn-xs" ng-click="addSemester(year)" ng-if="::!readonly">
+        <button class="btn btn-success btn-xs add-semester" ng-click="addSemester(year)" ng-if="::!readonly">
             Add Semester
         </button>
-        <button class="btn btn-default btn-xs" ng-click="removeYear($index)" ng-if="::!readonly">
+        <button class="btn btn-success btn-xs remove-year" ng-click="removeYear($index)" ng-if="::!readonly">
```

Comment:
Could we make the class of this button 'btn-danger' like the delete buttons? Since deleting a Year is also pretty dangerous